app/models/financialDetails/DocumentDetail.scala

```diff
@@ -101,26 +85,40 @@ case class DocumentDetail(taxYear: String,
     else isPaid
   }
 
-  val isPartPaid: Boolean = outstandingAmount.getOrElse[BigDecimal](0) != originalAmount.getOrElse[BigDecimal](0)
+  def remainingToPayByChargeOrLpi: BigDecimal = {
```

Comment:
I am not sure if we really need to do any changes in this file/logic: this is based on the fields requested in the requirements;
Please provide justification for making these changes;

Reply:
same as above